tests/integration/components/supplementary-file-browser-test.js

```diff
@@ -67,16 +112,16 @@ test('has additional files', function(assert) {
     assert.equal(this.$('#rightArrow').length, 1);
     assert.equal(this.$('#downArrow').length, 1);
 
+    // TODO enable when when tests are fixed
```

Comment:
Yes, `wait` should work here. [This article](https://www.mutuallyhuman.com/blog/2016/01/22/component-integration-testing-in-ember) mentions specifically using `wait` for integration testing of components that have asynchronous operations going on under the hood. I tried it here, but I'm not sure the test is set up correctly otherwise, so asserts were still failing. For an example of using `wait` see: https://github.com/CenterForOpenScience/ember-preprints/commit/28208e449136c19ce7556fc17de97109179b03e1#diff-57c6d61e1c420bd038a684e4cb14f95fR554